### ISSUE TITLE
1.4.19: Orcania not found

### ISSUE THREAD

**Issue by nunotexbsd** [babelouest/yder#24]
1.4.18 buids fine but 1.4.19 can't find Orcania. I'm comparing CMakeLists.txt from both versions but I can't find problem:

(FreeBSD 13.2 amd64)

```
===>   yder-1.4.19 depends on package: orcania>=2.3.0 - found
===>   yder-1.4.19 depends on file: /usr/local/bin/cmake - found
===>   yder-1.4.19 depends on executable: ninja - found
===>   yder-1.4.19 depends on shared library: libjansson.so - found (/usr/local/lib/libjansson.so)
===>   yder-1.4.19 depends on shared library: liborcania.so - found (/usr/local/lib/liborcania.so)
===>  Configuring for yder-1.4.19
===>  Performing out-of-source build
/bin/mkdir -p /wrkdirs/usr/ports/devel/yder/work/.build
-- The C compiler identification is Clang 14.0.5
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for __GNU_LIBRARY__
-- Looking for __GNU_LIBRARY__ - not found
-- Looking for _GNU_SOURCE
-- Looking for _GNU_SOURCE - not found
CMake Error at CMakeLists.txt:198 (message):
  Orcania not found


-- Configuring incomplete, errors occurred!
*** Error code 1
```

**Comment by babelouest** [comment]
Hello,

You usually have error like this if you haven't installed the [latest](https://github.com/babelouest/orcania/releases/tag/v2.3.2) orcania version (v2.3.2), or because you have multiple versions of orcania installed in your system, i.e. one in `/usr/lib` and the other in `/usr/local/lib`.

If it's the first cause, then the error message should be more relevant. I'll make the change.

**Comment by nunotexbsd** [comment]
@babelouest

Hi,

Current orcania version is 2.3.1 and only these version is installed in testport jail.
I will update it and redo the tests.

I was hoping that `message( FATAL_ERROR "Orcania version required: ${ORCANIA_VERSION_REQUIRED} - version installed: ${ORCANIA_VERSION_STRING}")` tells me the requirement.

**Comment by nunotexbsd** [comment]
@babelouest

Hello!

Build is fixed with orcania 2.3.2
Next update I will take more atention on dependencies.

Thanks very much
